src/database/config/config.js

```diff
@@ -1,7 +1,7 @@
 import dotenv from 'dotenv';
 
 dotenv.config();
-const { DEV_DATABASE_URL, DATABASE_URL, TEST_DATABASE_URL, SSL } = process.env;
+const { DEV_DATABASE_URL, DATABASE_URL, SSL } = process.env;
```

Comment:
why did you remvoe the TEST_DATABASE_URL, while  you can change the environment variable

Reply:
Didas suggested that we use a local database running in the container on circleci.